scanners/android/icon.go

```diff
@@ -0,0 +1,138 @@
+package android
+
+import (
+	"fmt"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/beevik/etree"
+	"github.com/bitrise-io/bitrise-init/models"
+	"github.com/bitrise-io/bitrise-init/utility"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/sliceutil"
+)
+
+type icon struct {
+	prefix       string
+	fileNameBase string
+}
+
+func lookupIconName(manifestPth string) ([]icon, error) {
+	doc := etree.NewDocument()
+	if err := doc.ReadFromFile(manifestPth); err != nil {
+		return nil, err
+	}
+
+	log.Debugf("Looking for app icons. Manifest path: %s", manifestPth)
+	parsedIcons, err := parseIconName(doc)
+	if err != nil {
+		return nil, err
+	}
+
+	return parsedIcons, nil
+}
+
+// parseIconName fetches icon name from AndroidManifest.xml.
+func parseIconName(doc *etree.Document) ([]icon, error) {
+	man := doc.SelectElement("manifest")
+	if man == nil {
+		log.Debugf("Key manifest not found in manifest file")
+		return nil, nil
+	}
+	app := man.SelectElement("application")
+	if app == nil {
+		log.Debugf("Key application not found in manifest file")
+		return nil, nil
+	}
+	ic := app.SelectAttr("android:icon")
+	if ic == nil {
+		log.Debugf("Attribute not found in manifest file")
+		return nil, nil
+	}
+
+	iconPathParts := strings.Split(strings.TrimPrefix(ic.Value, "@"), "/")
```

Comment:
NOTE: would be great to have a comment with a usual icon value, to see, why you parse the path this way

Reply:
`@mipmap/ic_launcher`